### PR TITLE
Debian client updates

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Updates to the Debian client content in the Client Cfg. Gives it a seperate top level page in the Client Cfg nav, and adds other arches and fixes order in the supported client matrix.

Drive-by edit: updates `{x86}_64` to the correct entity of `{x86_64}`.

Reported by @Shirocco88

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [ ] Manager-4.2
- [x] Uyuni

# Links

Fixes:

* https://github.com/uyuni-project/uyuni-docs/issues/461
* https://github.com/uyuni-project/uyuni-docs/issues/462

### CODE REVIEW

File: modules/client-configuration/pages/registration-overview-debian.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[[debian-registration-overview]]
+= Debian Client Registration
+
+You can register {debian} clients to your {productname} Server.
+The method and details varies depending on the operating system of the client.
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-overview-debian.adoc:5]: I do not understand this.  At this point it seems clear that you are talking about Debian.

File: modules/client-configuration/pages/supported-features.adoc [hunk 2]
```diff
@@ -41,114 +41,114 @@ The operating system you run on a client is supported by the organization that s
 | Salt Clients
 
 | {sle} 15
-| {x86}_64, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc}, {ibmz}, ARM
```
Comment (keichwa) [inline modules/client-configuration/pages/supported-features.adoc:44]: Changes in this file seem unrelated to Debian client updates ;)

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/supported-features.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/client-configuration/pages/supported-features.adoc:44]: yeah, that's why I noted it as a drive-by edit in the description :wink:

File: modules/client-configuration/pages/registration-overview-debian.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/client-configuration/pages/registration-overview-debian.adoc:5]: It's what all of them say. I see your point, but it would need to be edited globally.

**Review by jcayouette — APPROVED** [review]
